play-json-joda/src/main/scala/play/api/libs/json/JodaWrites.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+object JodaWrites extends JodaWrites
+
+trait JodaWrites {
+
+  /**
+   * Serializer for DateTime
+   * @param pattern the pattern used by SimpleDateFormat
```

Comment:
This uses DateTimeFormat, not SimpleDateFormat http://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html says "mostly compatible"